### ISSUE TITLE
ipv6, router advertisement ('ra') contains too much data

### ISSUE THREAD

**Issue by meirumeiru** [systemd/systemd#19762]
As far as I have understood the behaviour and the code, it seems to me, that the system is always sending all the routing and prefix information it knows, including those obtained from other ra packages. And this is wrong (at least that's how I read the rfc4861).

The information comming from an other routers ra package should not be included, when creating its own ra package.

I guess, that this behaviour is also responsible for the "strange behaviour" that many reported and the reason why accept_ra=2 cannot be used as default value (which I would say should be).

**Comment by yuwata** [comment]
> including those obtained from other ra packages.

Hm? I do not think so. Could you provide relevant .network files and debugging logs?

**Comment by meirumeiru** [comment]
I've seen conflicts with the configuration of my Windows server when using a VyOS router (based on this kernel) and a Windows router (RRAS) in the network after setting accept_ra=2. (The routing tables of the clients in the network didn't contain the information of the Windows router anymore.)

After removing the RRAS again from the network, I remarked, that the VyOS router was still distributing the information that was originally in the ra package of the Windows router. I concluded, that this was the cause of the conflict. (If I remember correctly, the distribution contained the prefix of the Windows router and the router ip of the VyOS router.)

But, to be honest, I don't have wireshark logs. I could set up a test network and to verify this theory and to get some network traffic logs.
